wurst/util/UnitUtils.wurst

```diff
@@ -0,0 +1,120 @@
+package UnitUtils
+import BuffObjEditing
+import ClosureTimers
+
+
+
+/*
+
+This file contains various utility functions for Unit, which require enhanced functionality like closures and cannot be defined in Unit.wurst
+
+*/
+
+@configurable public constant real FIELD_OF_VIEW_FRONT  = 115.00  // Real FoV is: 180 - 2x (FIELD_OF_VIEW-90)	Using a value of 115 generates a FoV of 130   (natural FoV is about 135 degrees)
+@configurable public constant real FIELD_OF_VIEW_BEHIND = 115.00 
```

Comment:
why is this a constant and not a function parameter?

Reply:
well cause its mostly not needed to be changed, could be seen as some default value that makes sense in most common situations, but just in case someone wants to use different settings, could be parameter and the 115 added into the hotdoc i think